lib/active_admin/resource.rb

```diff
@@ -1,4 +1,6 @@
 require 'active_admin/resource/action_items'
+require 'active_admin/resource/attributes'
+require 'active_admin/resource/ordering'
```

Comment:
@varyonic , attributes != ordering ... , don't understand what duplication do you mean